adafruit_bluefruit_connect/accelerometer_packet.py

```diff
@@ -0,0 +1,42 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 Dan Halbert for Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_bluefruit_connect.accelerometer_packet`
+====================================================
+
+Bluefruit Connect App Accelerometer data packet.
+
+* Author(s): Dan Halbert for Adafruit Industries
+
+"""
+
+from ._xyz_packet import _XYZPacket
+
+class AccelerometerPacket(_XYZPacket):
+    """A packet of x, y, z float values from an accelerometer."""
+
+    # Everything else is handled by _XYZPacket.
+    _TYPE_HEADER = b'!A'
+
+
+# Register this class with the superclass. This allows the user to import only what is needed.
+AccelerometerPacket.register_packet_type()
```

Comment:
I love the registration pattern. In the long term we probably want to match CPython which tracks subclasses by default: https://stackoverflow.com/questions/3862310/how-to-find-all-the-subclasses-of-a-class-given-its-name/3862957#3862957

Reply:
I didn't know about `SomeClass.__subclasses__`. Nice and would be even easier!